hgvs/config.py

```diff
@@ -65,7 +70,10 @@ def __getattr__(self, k):
         # Work around PyCharm bug https://youtrack.jetbrains.com/issue/PY-4213
         if k == "_cp":
             return
-        return ConfigGroup(self._cp[k])
+        try:
+            return ConfigGroup(self._cp[k])
+        except KeyError:
+            raise AttributeError(k)
```

Comment:
This broke where the test collector was calling `hasattr` on a config object. In python 2, hasattr returns False if _any_ exception is raised, while python 3 only catches `AttributeError`.